tool/create-exercise

```diff
@@ -105,6 +207,11 @@ Future main(args) async {
   await new File("${exerciseDir.path}/test/${filename}_test.dart").writeAsString(testTemplate(name));
   await new File("${exerciseDir.path}/pubspec.yaml").writeAsString(pubTemplate(name));
 
+  if (arguments["spec-path"] != null) {
+    // The output from test generation is not always well-formatted, use dartfmt to clean it up
+    await Process.run("dartfmt", ["-l", "120", "-w", "${exerciseDir.path}/test/${filename}_test.dart"]);
+  }
```

Comment:
Should include a case if formatter cannot format because something like `"\"Five quacking Zephyrs jolt my wax bed.\""` got turned into `""Five quacking Zephyrs jolt my wax bed.""` which will cause issues.

It would also help with debugging the create-exercise tool faster versus looking through the code for issues or having to re-run the formatter.